### DISCUSSION TITLE
Discontinuation of QuickReply

### DISCUSSION THREAD

**Discussion by Accio1** [theforumhelpers/theforumhelpers.github.io#325] [Announcements]
Due to lack of usage, development inactivity, and promoting canned responses which may not be very constructive QuickReply will be discontinued over the next two weeks.

### Timeline:
July 17th - This announcement is posted.
July 20th - References to QuickReply are removed from the main site.
July 23rd - GitHub Pages is disabled for QuickReply.
July 25th - The QuickReply repository is archived and made read-only.

### The Future of QuickReply
Part of the reason why QuickReply is being discontinued is because it promotes canned, potentially unconstructive responses to topics. Additionally, it is difficult to add responses because often times, there is not a one size fits all response that can be used on the forums.

Here are some possibilities for the future of QuickReply:
1. It becomes integrated with TFH Dashboard, allowing users to save their own responses to be easily copied.
2. It is split into multiple smaller tools. For example, a TOLORS quoter or a duplicate topic finder.
3. Feel free to share other ideas about how QuickReply can be modified to better suit the needs of forumers.

If this decision is heartbreaking for you, feel free to protest it, but if no one objects, QuickReply will be discontinued according to the timeline above. The code for QuickReply will remain available for people who want to use it to make their own version.

@leahcimto

**Comment by penguinmoose** [comment]
I don't mind QuickReply getting discontinued (I don't even use it).

**Comment by Accio1** [comment]
As of July 25th, 2022, QuickReply has been archived meaning that it is view only. It may be revisited in the future as mentioned in the main post.

**Reply by penguinmoose** [reply]
Ok. No problem with me

**Reply by Weredime** [reply]
July*
